lua/usip_player.lua

```diff
@@ -2527,20 +2553,16 @@ elseif tArgs.fCommandGetUidSelected then
         strExecReturnPath
     )
 
-    if not fFinalResult then
-        os.exit(1)
-    else
-        os.exit(0)
-    end
 
 --------------------------------------------------------------------------
 -- VERIFY CONTENT
 --------------------------------------------------------------------------
-elseif tArgs.fCommandVerifySelected then
+elseif tArgs.fCommandVerifySelected or tArgs.fCommandCheckSIPCookie then
```

Comment:
I'd make a new section for check SIP cookie, most of this block is specific to it anyway.
As a message, I'd use "CHECKING FOR SIP COOKIE"